### PR TITLE
Feature#23 nav bar design

### LINKED ISSUE #23: NavBar Design
# REQUIREMENTS
- NavBar design similar to mockup

# Acceptance Criteria
List all required acceptance criteria
- [ ] NavBar displays when you are sign in

### PR DESCRIPTION
**Pull request by AngelBFdev**
# Description
- NavBar design completed
- Seeker label returns to Feed
- Logout label Logs out
  
  ## Screenshots 
Mockup
image
Frontend
image

### CODE REVIEW

**Review by LuisGomez89 — APPROVED** [review]

File: app/assets/images/search.svg [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```
Comment (JuanequeX) [inline app/assets/images/search.svg]: What is the reason of this lines? xd

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -21,3 +22,9 @@ html {
 *:after {
   box-sizing: inherit;
 }
+
+.contenedor {
+  width: 95%;
+  max-width: 120rem;
+  margin: 0 auto;
```
Comment (JuanequeX) [inline app/assets/stylesheets/application.scss]: sort this block

File: app/assets/stylesheets/components/_navbar.scss [hunk 3]
```diff
@@ -0,0 +1,47 @@
+.header-nav {
+	padding-left: 1rem;
+	align-content: center;
+	background-color: $header_color;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_navbar.scss]: Sort this block

File: app/assets/stylesheets/components/_navbar.scss [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+	.bar{
+		font-size: 2rem;
+		display: flex;
+		padding: 1.5rem;
+		gap: 1rem;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_navbar.scss]: sort this block

File: app/assets/stylesheets/components/_navbar.scss [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+		a{
+			text-decoration: none;
+			color: $color_white;
+		}
+
+		.input_search{
+			width: 50%;
+			margin-left: 3rem;
+			background-color: $background_color;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_navbar.scss]: sort this block

File: app/assets/stylesheets/components/_navbar.scss [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+		}
+
+		.logout{
+			font-size: 2.5rem;
+			margin-left: auto;
+		}
+
+		a{
+			text-decoration: none;
+			color: $color_white;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_navbar.scss]: sort this block

File: app/assets/stylesheets/components/_navbar.scss [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+		}
+
+		.button_search {
+			@extend .input_search;
+			width: 7rem;
+			background: url(search.svg);
+			background-repeat: no-repeat;
+			background-color: $background_color;
+			background-position: center center;
+			background-size: 3.5rem;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_navbar.scss]: sort this line

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: app/assets/images/search.svg (hunk 1 above)
Comment (YilKanda) [inline app/assets/images/search.svg]: I ask the same question as @JuanequeX

File: app/assets/stylesheets/components/_navbar.scss [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+			@extend .input_search;
+			background-color: $background_color;
+			background-position: center center;
+			background-repeat: no-repeat;
+			background-size: 3.5rem;
+			background: url(search.svg);
+			width: 7rem;
+		}
+	}
+}
```
Comment (YilKanda) [inline app/assets/stylesheets/components/_navbar.scss]: Check the indentation of your file.

Comment (JuanequeX) [reply app/assets/stylesheets/components/_navbar.scss]: @YilKanda , i think it is a problem with github, logically the indentation in this file works well, can test it review the spaces with your cursor, another posibility is that they can have in vs code settings the tab sizes in 4 spaces even in this block with indentation in 2 spaces git takes the another configutarion

File: app/assets/images/search.svg (hunk 1 above)
Comment (AngelBFdev) [reply app/assets/images/search.svg]: We have no idea, we just pasted the svg

Comment (AngelBFdev) [reply app/assets/images/search.svg]: We have no idea, it was with the svg when we pasted

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (AngelBFdev) [reply app/assets/stylesheets/application.scss]: Solve

File: app/assets/stylesheets/components/_navbar.scss (hunk 3 above)
Comment (AngelBFdev) [reply app/assets/stylesheets/components/_navbar.scss]: Solve

File: app/assets/stylesheets/components/_navbar.scss (hunk 4 above)
Comment (AngelBFdev) [reply app/assets/stylesheets/components/_navbar.scss]: Solve

File: app/assets/stylesheets/components/_navbar.scss (hunk 5 above)
Comment (AngelBFdev) [reply app/assets/stylesheets/components/_navbar.scss]: Solve

File: app/assets/stylesheets/components/_navbar.scss (hunk 6 above)
Comment (AngelBFdev) [reply app/assets/stylesheets/components/_navbar.scss]: Solve

File: app/assets/stylesheets/components/_navbar.scss (hunk 7 above)
Comment (AngelBFdev) [reply app/assets/stylesheets/components/_navbar.scss]: Solve

File: app/assets/stylesheets/components/_navbar.scss (hunk 8 above)
Comment (AngelBFdev) [reply app/assets/stylesheets/components/_navbar.scss]: I don't understand why does it look like this here, in the file we have 2 spaces for identation and we don't know how to solve this

File: app/assets/images/search.svg (hunk 1 above)
Comment (JuanequeX) [reply app/assets/images/search.svg]: this lines even are not should necessary to a svg works, review how is the reason and fixed it

File: app/assets/stylesheets/components/_navbar.scss (hunk 8 above)
Comment (AngelBFdev) [reply app/assets/stylesheets/components/_navbar.scss]: Solved

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by IrenehMDev — COMMENTED** [review]
LGTM

**Review by IrenehMDev — APPROVED** [review]
LGTM

**Pull request merged**